### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**ARCHIVE SCREEN**

- Archive page
- Archive title
- bg color change

### CODE REVIEW

File: pages/home.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.note-container {
+  margin: 2rem 5rem;
+}
+.wrapper {
+  display: flex;
+  padding: 2rem 0rem;
+  justify-content: space-between;
+}
+aside {
```
Comment (AmishMishra11) [inline pages/home.css:9]: dont use elements directly make a class and then apply CSS

File: pages/home.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  color: var(--primary-color);
+  cursor: pointer;
+  transition: all 0.3s;
+  font-weight: bolder;
+}
+.note-btn:hover {
+  background-color: var(--primary-color);
+  color: var(--gradient-color);
+}
+ul li {
```
Comment (AmishMishra11) [inline pages/home.css:26]: same as above

**Review by AmishMishra11 — COMMENTED** [review]
change the method to apply css

**Review by deepak29-git — APPROVED** [review]

File: pages/home.css (hunk 1 above)
Comment (tanishk15sharma) [reply pages/home.css:9]: okay I will change this .

File: pages/home.css (hunk 2 above)
Comment (tanishk15sharma) [reply pages/home.css:26]: sure

**Pull request merged**